vm/src/vm/vm_new.rs

```diff
@@ -270,7 +270,9 @@ impl VirtualMachine {
         } else {
             Vec::new()
         };
-        self.new_exception(self.ctx.exceptions.stop_iteration.to_owned(), args)
+        let exc = self.new_exception(self.ctx.exceptions.stop_iteration.to_owned(), args.clone());
+        // To initialize the `value` attribute to first argument
+        self.set_exception_attr(exc, "value", args.get(0).cloned())
```

Comment:
I wanted this code is showing it is same to `StopIteration.__init__`, but it still doesn't look like it.